talons:
    back_left:
        id: 03
        inverted: False
    back_right:
        inverted: True
        id: 02
    front_left:
        inverted: False
        id: 01
    front_right:
        inverted: True
        id: 04

